docs/pages/libraries/clis.md

```diff
@@ -15,6 +15,7 @@ parent: Recommended libraries
 
 ### Other useful tools for CLIs
 
-| Name                            | Short description       | 🚦  |
-| ------------------------------- | ----------------------- | :-: |
-| [tqdm](https://tqdm.github.io/) | Progress bar for loops. | 🟢  |
+| Name                                                     | Short description                                              | 🚦  |
+| -------------------------------------------------------- | -------------------------------------------------------------- | :-: |
+| [tqdm](https://tqdm.github.io/)                          | Progress bar for loops.                                        | 🟢  |
+| [inquirer] (<https://github.com/magmax/python-inquirer>) | Collection of common interactive command line user interfaces. | 🟢  |
```

Comment:
```suggestion
| Name                                                     | Short description                                              | 🚦  |
| ------------------------------------------------------ | -------------------------------------------------------------- | :-: |
| [tqdm](https://tqdm.github.io/)                        | Progress bar for loops.                                        | 🟢  |
| [inquirer] (https://github.com/magmax/python-inquirer) | Collection of common interactive command line user interfaces. | 🟢  |
```

Reply:
Bit confused  - was @samcunliffe's suggested change accepted? This is marked as resolved but it looks like the Markdown syntax is still not quite correct here to me. Should this not be either `[inquirer](https://github.com/magmax/python-inquirer)` so this displays as 

> [inquirer](https://github.com/magmax/python-inquirer) 

or potentially `inquirer (<https://github.com/magmax/python-inquirer>)` if the intention was to have the URL explicitly included in parentheses - that is 

> inquirer (<https://github.com/magmax/python-inquirer>)

Reply:
```suggestion
| Name                                                  | Short description                                              | 🚦  |
| ----------------------------------------------------- | -------------------------------------------------------------- | :-: |
| [tqdm](https://tqdm.github.io/)                       | Progress bar for loops.                                        | 🟢  |
| [inquirer](https://github.com/magmax/python-inquirer) | Collection of common interactive command line user interfaces. | 🟢  |
```

Reply:
🙈 thanks for catching, I should have fixed the @samcunliffe - pre-commit commit loop with the latest commit.